src/enter.js

```diff
@@ -26,6 +26,6 @@ export default class Enter extends Feature {
 		this.listenTo( editingView, 'enter', ( evt, data ) => {
 			editor.execute( 'enter' );
 			data.preventDefault();
-		} );
+		}, 'low' );
```

Comment:
In engine, many callbacks are added with `low` priority hinting, that these are default callbacks that needs to be overwritten. In a result, most callbacks added in (external) plugins/features can be added with default priority and still overwrite default ones.

So, I felt like this is a very basic functionallity, almost like those in engine, hence I decided to go with `low` priority. If you really think this is wrong, I will change it. But remember that we have only 5 levels of events, so users will only get `high` and `highest`.